opennms-base-assembly/src/main/filtered/etc/snmp-graph.properties.d/juniper-graph.properties

```diff
@@ -24,6 +28,108 @@ ive.connections
+ --vertical-label="percent" \
+ --lower-limit 0 \
+ --upper-limit 105 \
+ DEF:dpercent={rrd1}:jnxOpCpu:AVERAGE \
+ CDEF:dpercent10=0,dpercent,GT,0,dpercent,IF \
+ CDEF:dpercent20=10,dpercent,GT,0,dpercent,IF \
+ CDEF:dpercent30=20,dpercent,GT,0,dpercent,IF \
+ CDEF:dpercent40=30,dpercent,GT,0,dpercent,IF \
+ CDEF:dpercent50=40,dpercent,GT,0,dpercent,IF \
+ CDEF:dpercent60=50,dpercent,GT,0,dpercent,IF \
+ CDEF:dpercent70=60,dpercent,GT,0,dpercent,IF \
+ CDEF:dpercent80=70,dpercent,GT,0,dpercent,IF \
+ CDEF:dpercent90=80,dpercent,GT,0,dpercent,IF \
+ CDEF:dpercent100=90,dpercent,GT,0,dpercent,IF \
+ COMMENT:"\\n" \
+ COMMENT:"CPU usage graph colors\\n" \
+ AREA:dpercent10#5ca53f:" 0-10%" \
+ AREA:dpercent20#75b731:"11-20%" \
+ AREA:dpercent30#90c22f:"21-30%" \
+ AREA:dpercent40#b8d029:"31-40%" \
+ AREA:dpercent50#e4e11e:"41-50%" \
+ COMMENT:"\\n" \
+ AREA:dpercent60#fee610:"51-60%" \
+ AREA:dpercent70#f4bd1b:"61-70%" \
+ AREA:dpercent80#eaa322:"71-80%" \
+ AREA:dpercent90#de6822:"81-90%" \
+ AREA:dpercent100#d94c20:"91-100%" \
+ COMMENT:"\\n" \
+ LINE1:dpercent#46683b:"CPU usage in %" \
+ GPRINT:dpercent:AVERAGE:"Avg \\: %7.3lf%s" \
+ GPRINT:dpercent:MIN:"Min \\: %7.3lf%s" \
+ GPRINT:dpercent:MAX:"Max \\: %7.3lf%s\\n"
+
+report.juniper.jnxOpTableLoad.name=Juniper Load Average
+report.juniper.jnxOpTableLoad.columns=jnxOpLoad1Min, jnxOpLoad5Min, jnxOpLoad15Min
+report.juniper.jnxOpTableLoad.type=jnxOperatingTable
+report.juniper.jnxOpTableLoad.propertiesValues=jnxOpDescr
+report.juniper.jnxOpTableLoad.command=--title="Juniper: Load Average on {jnxOpDescr}" --units-exponent=0 \
+ DEF:avg1={rrd1}:jnxOpLoad1Min:AVERAGE \
+ DEF:minAvg1={rrd1}:jnxOpLoad1Min:MIN \
+ DEF:maxAvg1={rrd1}:jnxOpLoad1Min:MAX \
+ DEF:avg5={rrd2}:jnxOpLoad5Min:AVERAGE \
+ DEF:minAvg5={rrd2}:jnxOpLoad5Min:MIN \
+ DEF:maxAvg5={rrd2}:jnxOpLoad5Min:MAX \
+ DEF:avg15={rrd3}:jnxOpLoad15Min:AVERAGE \
+ DEF:minAvg15={rrd3}:jnxOpLoad15Min:MIN \
+ DEF:maxAvg15={rrd3}:jnxOpLoad15Min:MAX \
+ CDEF:float1=avg1,100,/ \
+ CDEF:minFloat1=minAvg1,100,/ \
+ CDEF:maxFloat1=maxAvg1,100,/ \
+ CDEF:float5=avg5,100,/ \
+ CDEF:minFloat5=minAvg5,100,/ \
+ CDEF:maxFloat5=maxAvg5,100,/ \
+ CDEF:float15=avg15,100,/ \
+ CDEF:minFloat15=minAvg15,100,/ \
+ CDEF:maxFloat15=maxAvg15,100,/ \
+ AREA:float1#babdb6:"1  minute" \
+ GPRINT:float1:AVERAGE:"Avg \\: %10.2lf" \
+ GPRINT:float1:MIN:"Min \\: %10.2lf" \
+ GPRINT:float1:MAX:"Max \\: %10.2lf\\n" \
+ AREA:float5#888a85:"5  minute" \
+ GPRINT:float5:AVERAGE:"Avg \\: %10.2lf" \
+ GPRINT:float5:MIN:"Min \\: %10.2lf" \
+ GPRINT:float5:MAX:"Max \\: %10.2lf\\n" \
+ LINE2:float15#a40000:"15 minute" \
+ GPRINT:float15:AVERAGE:"Avg \\: %10.2lf" \
+ GPRINT:float15:MIN:"Min \\: %10.2lf" \
+ GPRINT:float15:MAX:"Max \\: %10.2lf\\n"
+
+report.juniper.jnxOpTableBuff.name=Juniper Buffer Utilization
+report.juniper.jnxOpTableBuff.columns=jnxOpBuff
+report.juniper.jnxOpTableBuff.type=jnxOperatingTable
+report.juniper.jnxOpTableLoad.propertiesValues=jnxOpDescr
```

Comment:
Hi @indigo423 

I overlooked the following typo in jnxOpTableBuff graph:

+report.juniper.jnxOpTableBuff.name=Juniper Buffer Utilization
+report.juniper.jnxOpTableBuff.columns=jnxOpBuff
+report.juniper.jnxOpTableBuff.type=jnxOperatingTable
+report.juniper.**jnxOpTableLoad**.propertiesValues=jnxOpDescr

Could you fix it quickly? 

Sorry for the inconvenience

Cyr